mackup/applications/secure-pipes.cfg

```diff
@@ -0,0 +1,9 @@
+[application]
+name = Secure Pipes
+
+[configuration_files]
+Library/Preferences/net.edgeservices.connections.plist
+Library/Preferences/net.edgeservices.Secure-Pipes.plist
+Library/Preferences/net.edgeservices.sp-config.plist
+Library/Application Support/Secure Pipes/*.known_hosts
```

Comment:
Wildcards are not supported, Mackup link files and directories, and you can't link a wildcard.